include/pyclient.h

```diff
@@ -401,12 +403,12 @@ class PyClient
         *          specified frequency for a specified number
         *          of times. The name will be automatically prefixed
         *          base on prefixing behavior.
-        *   \param name The key that will be checked in the database
+        *   \param name The name that will be checked in the database
         *   \param poll_frequency_ms The frequency of checks for the
-        *                            key in milliseconds
+        *                            name in milliseconds
         *   \param num_tries The total number of times to check for
-        *                    the specified number of keys.
-        *   \returns Returns true if the key is found within the
+        *                    the specified number of names.
```

Comment:
I think this should be ``The total number of times to check for the model or script.``